src/backend/api/meals.js

```diff
@@ -1,16 +1,165 @@
-const express = require("express");
+const express = require("express")
 const router = express.Router();
 const knex = require("../database");
 
+
+
+
+
+//  /api/meals/:meal_id/reviews	GET	Returns all reviews for a specific meal.
+//  Reviews
+
+router.get("/:meal_id/reviews", async (request, response) => {
+
+
+  let mealReviews = await knex("Review").select().where({ "Review.meal_id": request.params.meal_id })
+
+
+  if (mealReviews.length > 0) {
+    response.json(mealReviews)
+  } else {
+    response.status(404).json({ error: "Review Not found" })
+  }
+})
+
+
+
+
 router.get("/", async (request, response) => {
-  try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const meal = await knex("meals").select();
-    response.json(meal);
 
-  } catch (error) {
-    throw error;
+  // maxPrice	Number	Returns all meals that are cheaper than maxPrice.	api/meals?maxPrice=90
+
+  if ("maxPrice" in request.query) {
+    if (isNaN(request.query.maxPrice)) {
+
+      response.status(404).json({ error: "max price is not a number" });
```

Comment:
You're missing a `return` after this line to avoid the function to continue further down.